crates/polars-io/src/csv/read/read_impl/batched.rs

```diff
@@ -76,27 +92,12 @@ impl Iterator for ChunkOffsetIter<'_> {
                 if self.last_offset == self.bytes.len() {
                     return None;
                 }
-                let bytes_first_row = if self.rows_per_batch > 1 {
-                    let bytes_first_row = next_line_position(
-                        &self.bytes[self.last_offset + 2..],
-                        Some(self.expected_fields),
-                        self.separator,
-                        self.quote_char,
-                        self.eol_char,
-                    )
-                    .unwrap_or(1);
-                    bytes_first_row + 2
-                } else {
-                    1
-                };
                 get_file_chunks_iterator(
                     &mut self.offsets,
                     &mut self.last_offset,
                     self.n_chunks,
-                    self.rows_per_batch * bytes_first_row,
+                    &mut self.chunk_size,
```

Comment:
I've currently changed the (initial) chunk size to the default value copied from `read_impl` from the in-memory CSV reader, and ignoring the user-provided `batch_size` option.

Reply:
Yes, good one. I benchmarked that one. I think that one is a good cache friendly default.